.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,70 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty app.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Have a question? 👉 [Start a new discussion in Discord](https://crpt.fyi/berty-discord)
```

Comment:
Can you remove this for now? We may add a `Question` issue template on Github that will be managed by @jefft0. We'll see if we add a link to the Discord after that.